cmd/serve/main.go

```diff
@@ -369,6 +369,13 @@ func claimsMiddleware(conf *config.AppConfig) authsession.ClaimsMiddleware {
 		if (user.Provider == "Basic" || user.Provider == "Simple") && user.Subject == conf.AdminUsername {
 			user.Claims.Add("admin", "true")
 		}
+		// allow access to users only when access claim is present for OIDC
+		if user.Provider == conf.Auth.OIDC.Name && conf.Auth.OIDC.AccessClaim != "" {
```

Comment:
```suggestion
		if conf.Auth.OIDC != nil && user.Provider == conf.Auth.OIDC.Name && conf.Auth.OIDC.AccessClaim != "" {
```

`conf.Auth.OIDC` is `nil` if OIDC is disabled -> 
```
runtime error: invalid memory address or nil pointer dereference
  github.com/freifunkMUC/wg-access-server/cmd/serve.claimsMiddleware.func1(0xc0002f6060)
    wg-access-server/cmd/serve/main.go:373
```